PhysicsTools/Heppy/python/analyzers/core/AutoHandle.py

```diff
@@ -7,12 +7,13 @@ class AutoHandle( Handle, object ):
 
     handles = {}
     
-    def __init__(self, label, type, mayFail=False, fallbackLabel=None):
+    def __init__(self, label, type, mayFail=False, fallbackLabel=None, lazy=True):
         '''Note: label can be a tuple : (module_label, collection_label, process)'''
         self.label = label
         self.fallbackLabel = fallbackLabel
         self.type = type
         self.mayFail = mayFail
+	self.lazy = lazy
```

Comment:
I haven’t checked in a text editor, just on github, so it could be that it’s a display bug in github. 
I’m using spaces with automatic indentation in emacs. 

Colin



Le 28 mai 2015 à 16:24, Giovanni Petrucciani <notifications@github.com> a écrit :

> In PhysicsTools/Heppy/python/analyzers/core/AutoHandle.py:
> 
> >          '''Note: label can be a tuple : (module_label, collection_label, process)'''
> >          self.label = label
> >          self.fallbackLabel = fallbackLabel
> >          self.type = type
> >          self.mayFail = mayFail
> > +	self.lazy = lazy
> Look ok to me on linux, but ok, can fix. What are you using in heppy, spaces or tabs? Giovanni Il 28-mag-2015 16:06, "Colin Bernet" <notifications@github.com> ha scritto:
> …
> —
> Reply to this email directly or view it on GitHub.
>